defaults/main.yml

```diff
@@ -41,16 +41,67 @@ docker__distribution_release: '{{ ansible_local.core.distribution_release
 docker__upstream: False
 
                                                                    # ]]]
+# .. envvar:: docker__variant [[[
+#
+# For upstream repositories the variant to be installed: ce or ee. Note that Docker EE
+# is not supported on Debian.
+docker__variant: 'ce'
+
+                                                                   # ]]]
+# .. envvar:: docker__channel [[[
+#
+# For upstream repositories choose the stable or edge channel.
+docker__channel: 'stable'
```

Comment:
I would rename the above variables to `docker__upstream_variant` (is that official name, or perhaps "flavor" would be better?) and `docker__upstream_channel` to better indicate their purpose.